flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandler.java

```diff
@@ -291,6 +290,11 @@ private void decodeMsg(Object msg) throws Throwable {
 					}
 				}
 			}
+		} else if (msgClazz == NettyMessage.AddBacklog.class) {
```

Comment:
nit: import `NettyMessage.AddBacklog` because it occurs many times in this part.